packages/panzer/core/cmake/PanzerCore_config.hpp.in

```diff
@@ -22,13 +22,8 @@ namespace panzer {
   // long" over "int". This code will choose long long over int. When
   // tpetra removes deprecated code, we can just default directly to
   // tpetra.
-#if defined(HAVE_TPETRA_INST_INT_LONG_LONG)
-  using GlobalOrdinal = long long int;
-#else
-  using GlobalOrdinal = Tpetra::Details::DefaultTypes::global_ordinal_type;
-#endif
-
-  using LocalOrdinal = int;
```

Comment:
Is this commit removing all the deprecated code in tpetra? If not, changes to this file will cause failures in downstream apps when deprecated code is enabled. If multiple GOs are enabled in Trilinos, we need the the GO to always prefer long long over other types. In deprecated code the default chooses int over long long while for non-deprecated code, it does choose long long first. Also - the use of DefaultTypes was so that we don't include and have to parse a ton of tpetra code that is never used (slowing down build times). All we want is the GO type. @mhoemmen plans to move the Tpetra_Details_DefaultTypes.hpp out of Details namespace and expose to users for this use case. We just jumped the gun here. I would prefer to leave this file as is.

Reply:
@rppawlo - this PR is not going to remove deprecated code, I was just trying to get a build that worked with deprecated code disabled.  Any file that failed to build with deprecated code disabled, I applied the `Tpetra::Map<>::global_ordinal_type` bandaid.  This PR still has many build and test failures (with deprecated code off), I’ll revisit this file.

Reply:
> @mhoemmen plans to move the Tpetra_Details_DefaultTypes.hpp out of Details namespace and expose to users for this use case.

Rather, "@trilinos/tpetra plans to move ...." :-)

Reply:
The original code looks ok, but failed to build.  Let me see if I can find the actual build error that led me to make this change (it would t build otherwise)